src/collection/collection.controller.ts

```diff
@@ -94,11 +94,13 @@ export class CollectionController {
   @UseGuards(BrokerOidcAuthGuard)
   async generateAccountToken(
     @Param('id') id: string,
+    @Query('expirationDaysInSeconds') expirationDaysInSeconds: number = 7776000,
```

Comment:
Required parameter. Should have no default. 'expirationDaysInSeconds' -> 'expiration'
Can add @ApiQuery({description: 'blah blah in seconds'}) to generateAccountToken